check-links/crawl-for-broken-links.sh

```diff
@@ -19,7 +19,6 @@ EXCLUDE_PATTERN="(http://|https://("
 while read -r; do
     EXCLUDE_PATTERN+="${REPLY}|"
 done<<EOF
-EXWPKK5PV4-dsn.algolia.net
```

Comment:
Intentional?

Reply:
i ran the checker locally and it completes